test/extended/clusterup.sh

```diff
@@ -293,26 +261,38 @@ readonly default_tests=(
     "svcaccess"
     "persistentvolumes"
 
-# enable once https://github.com/openshift/origin/issues/16995 is fixed
-#    "default"
-# enable once https://github.com/openshift/origin/issues/16995 is fixed
-#    "image::ose3.3"
-# enable once https://github.com/openshift/origin/issues/16995 is fixed
-#    "image::ose3.4"
-# enable once https://github.com/openshift/origin/issues/16995 is fixed
-#    "image::ose3.5"
-
-    "image::ose3.6"
-
 # logging+metrics team needs to fix/enable these tests.
 #    "metrics"
 #    "logging"
 )
 
+# run each test with each of these set of additional args.  Primarily
+# intended to run the tests against different cluster versions.
+readonly extra_args=(
+    # Test the previous OCP release
+    # TODO - enable this once v3.9 ships, v3.7 didn't have a TSB image so it's 
+    # annoying to test.
+    #"--loglevel=2 --image=registry.access.redhat.com/openshift3/ose --version=v3.7"
+
+    # Test the previous origin release
+    "--loglevel=2 --image=docker.io/openshift/origin --version=v3.7.0"
+
+    # Test the current published release
+    "--loglevel=2"  # can't be empty, so pass something benign
+
+    # Test the code being delivered
+    "--loglevel=2 --version=${ORIGIN_COMMIT}"
```

Comment:
ORIGIN_COMMIT shouldn't be used before a default is set

Reply:
yeah, thanks.  i didn't notice since i was only running this in the CI env that always set it.  I see you've addressed it in your PR.